### ISSUE TITLE
File config.envs.php

### ISSUE THREAD

**Issue by nunoms00** [fossjobs/fossjobs#68]
Hi,
Is this "config.envs.php" file missing from the config folder?

Thanks

**Comment by moba** [comment]
Indeed! You can find the original template at https://github.com/jobberbase/jobberbase/blob/master/public/_config/config.envs.php . In our deployment, it contains sensitive information. For the sake of completeness, we could add the template back to this repo. Pull requests welcome.
